src/App.tsx

```diff
@@ -1,14 +1,60 @@
 /* eslint-disable max-len */
```

Comment:
Consider removing the `/* eslint-disable max-len */` comment if you don't have any line exceeding the max length allowed by the eslint rules. If you still need this rule, it would be better to disable it specifically for the lines that need it instead of disabling it for the entire file.